scripts/validate_rpm_specs.py

```diff
@@ -39,12 +45,12 @@ def validate_rpm_spec(spec_file_path):
     """
 
     try:
+        LOG.info("Verifying file %s" %spec_file_path)
         run_command("rpmlint -f .rpmlint -v %s" % spec_file_path)
     except exception.SubprocessError as e:
         #pylint: disable=no-member
         print("validation of RPM specification file %s failed, output: %s" % (spec_file_path, e.stdout))
```

Comment:
Now that we have proper logging, you can replace this print with `LOG.exception`. This will print the exception stack trace.
The same should be done for all prints, using `LOG.error` if there's no exception involved.